portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,39 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// photos data
+import photos_data from './photos-data.js';
```

Comment:
Module imports should use `lowerCamelCase`. [See the style guide for naming conventions of imports and exports](https://google.github.io/styleguide/jsguide.html#es-module-imports).